src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -150,6 +150,21 @@
   end
 
   describe 'GET #users' do
+    before do
+      @project = create(:project)
+      create(:relationship_project_user, project: @project, user: create(:confirmed_user))
+      create(:relationship_project_user, project: @project, user: create(:confirmed_user))
```

Comment:
LoL! I can't use the create_list method because I need 2 different users related with the project